manubot/cite/handlers.py

```diff
@@ -222,83 +263,180 @@ def get_csl_item(self, citekey) -> Dict[str, Any]:
     "bionumbers": "manubot.cite.curie.Handler_CURIE",
     "bioportal": "manubot.cite.curie.Handler_CURIE",
     "bioproject": "manubot.cite.curie.Handler_CURIE",
+    "bioregistry": "manubot.cite.curie.Handler_CURIE",
+    "bioregistry.collection": "manubot.cite.curie.Handler_CURIE",
+    "bioregistry.schema": "manubot.cite.curie.Handler_CURIE",
+    "biorxiv": "manubot.cite.curie.Handler_CURIE",
```

Comment:
This is relevant to #282.

Is it right that support for the `chemrxiv` prefix is not being added here?

Reply:
Related: all of the chemrxiv stuff is broken since they switched from figshare to their new closed system

Reply:
It looks like `chemrxiv` is not prefix in bioregistry, so that seems correct for it not to be added here.